index.js

```diff
@@ -86,41 +63,124 @@ async function updateIssues(issues, newLabelId) {
       }
     }
 
-    issue.update(payload);
+    await issue.update(payload);
 
     core.info(
       `Moved "${issue.title}" from state ${fromStateId} to ${toStateId}.`
     );
-  });
+  }
 }
 
-const escapeText = (text) =>
-  text.replace(ESPACE_REGEX, (match) => ESCAPE[match]);
-
-const formatIssues = (issues) =>
-  issues
-    .map(({ title, url }) => `- <${url}|${escapeText(title)}>`)
-    .join("\n") || "_No tickets_";
-
 const formatIssuesForSlack = async (labelId) => {
   const label = await linearClient.issueLabel(labelId);
-  const bugs = await label.issues({
-    filter: { labels: { name: { eq: "Bug" } } },
+  const bugs = await getBugs(label);
+  const chores = await getChores(label);
+  const fastTracks = await getFastTracks(label);
+  const projects = await getProjects(label);
+
+  return `
+*Fast Track*
+${formatIssues(fastTracks)}
+
+*Bugfixes*
+${formatIssues(bugs)}
+
+*Chores*
+${formatIssues(chores)}
+
+*Projects*
```

Comment:
In the task it says to name the new section "Projects in progress", but I think it's more fitting to just call it "Projects" as it can also include completed projects.

Reply:
Ok, could we then add a way to distinguish whether a project was completed though in the release? So basically mark a project for which all issues have been completed in the release as complete.

Reply:
Done. Will look like this:

```
Projects
- Project A, Completed
- Project B, Progress: 66%
```

Reply:
🤩